src/tests/data/create-non-admin-user.yaml

```diff
@@ -0,0 +1,16 @@
+---
+Username: test2@gmail.com
+Roles:
+- User
+- SignupUser
+VpnPassword: "********"
```

Comment:
Same question as above, isn't `vpnpassword` readonly? Is this really part of the input?

Reply:
This was present in request body of the api call made from UI